angular/projects/spark-angular/src/lib/directives/inputs/sprk-button/sprk-button.stories.ts

```diff
@@ -1,19 +1,136 @@
 import { SprkButtonModule } from './sprk-button.module';
 
-export default { title: 'Components|Button' };
+export default {
+  title: 'Components|Buttons',
+  component: SprkButtonModule,
+ };
 
 const modules = {
-  imports: [SprkButtonModule],
+  imports: [
+    SprkButtonModule,
+  ],
 };
 
 export const primary = () => ({
   moduleMetadata: modules,
   template: `
     <button
-      data-id="button-1"
+      data-id="button-primary"
       sprkButton
     >
       Button
     </button>
   `,
 });
+
+export const secondary = () => ({
+  moduleMetadata: modules,
+  template: `
+    <button
+      class="sprk-c-Button--secondary"
+      data-id="button-secondary"
+      sprkButton
+    >
+      Button
+    </button>
+  `,
+});
+
+export const tertiary = () => ({
+  moduleMetadata: modules,
+  template: `
+    <button
+      class="sprk-c-Button--tertiary"
+      data-id="button-tertiary"
+      sprkButton
+    >
+      Button
+    </button>
+  `,
+});
+
+export const disabledPrimary = () => ({
+  moduleMetadata: modules,
+  template: `
+    <button
+      disabled
+      data-id="button-disabled-primary"
+      sprkButton
+    >
+      Button
+    </button>
+  `,
+});
+
+export const disabledSecondary = () => ({
+  moduleMetadata: modules,
+  template: `
+    <button
+      disabled
+      class="sprk-c-Button--secondary"
+      data-id="button-disabled-secondary"
+      sprkButton
+    >
+      Button
+    </button>
+  `,
+});
+
+export const disabledTertiary = () => ({
+  moduleMetadata: modules,
+  template: `
+    <button
+      disabled
+      class="sprk-c-Button--tertiary"
+      data-id="button-disabled-tertiary"
+      sprkButton
+    >
+      Button
+    </button>
+  `,
+});
+
+export const submissionSpinner = () => {
+  return {
+    moduleMetadata: modules,
+    template: `
+      <button
+        (click)="toDoSpinner()"
```

Comment:
this is the part that @RCopeland  will use for his task on removing the vanilla dep.